### PR TITLE
meson: Write the correct path to the data in config.h

### PR DESCRIPTION
**Pull request by andy5995**

### CODE REVIEW

**Comment by andy5995** [comment]
No clue why this fails.

**Comment by andy5995** [comment]
@winrid I've removed the 'meson install' so this should pass now. I have absolutely no idea why it errors with `Error executing command as another user: Not authorized` when trying to install.

Once this is merged, we can update the docker images so they'll be fixed as well.

**Comment by andy5995** [comment]
I opened a discussion https://github.com/mesonbuild/meson/discussions/12874 maybe @eli-schwartz will know, or maybe I need to ask on GitHub discussions.

File: src/NetPanzer/Core/main.cpp [hunk 1]
```diff
@@ -251,6 +251,8 @@ BaseGameManager* initialise(int argc, char** argv) {
     filesystem::addToSearchPath(dataDir.c_str());
   } else {
     LOGGER.warning("Data directory not specified! Game will only work if binary is in source root.");
+    if (!filesystem::exists("maps"))
```
Comment (andy5995) [inline src/NetPanzer/Core/main.cpp:254]: I think the fail data test didn't work because netpanzer wasn't exiting cleanly enough, so I added this.

Comment (andy5995) [reply src/NetPanzer/Core/main.cpp:254]: Nope, that didn't fix it...

**Comment by andy5995** [comment]
I believe this is working and I'm done with it.

File: .github/workflows/c-cpp.yml [hunk 2]
```diff
@@ -93,3 +96,15 @@ jobs:
       run: |
         cd _build
         meson test --suite=netpanzer -v
+
+    - name: Data install test
+      if: ${{ contains(matrix.build.name, 'data-install-test') }}
+      run: |
+        unset NETPANZER_DATADIR
+        cd _build
+        # want fail
```
Comment (andy5995) [inline .github/workflows/c-cpp.yml:105]: @winrid The "pass" and "fail" test in this section is now working as expected

**Review by winrid — APPROVED** [review]

**Pull request merged**